src/main/java/org/benetech/servicenet/service/impl/OrganizationMatchServiceImpl.java

```diff
@@ -150,11 +148,19 @@ public void createOrUpdateOrganizationMatches(Organization organization) {
 
     private List<OrganizationMatch> findAndPersistMatches(Organization organization, List<Organization> notMatchedOrgs) {
         List<OrganizationMatch> matches = new LinkedList<>();
+        long startTime = System.currentTimeMillis();
```

Comment:
please add TODO for removing this logic in the future, you cloud also add a link to an issue for refactoring the logs.